esp-metadata/devices/esp32s3.toml

```diff
@@ -75,8 +75,6 @@ symbols = [
     "assist_debug_region_monitor",
     "gdma",
     "phy",
-    "bt",
```

Comment:
These symbols are also used in the xtask, I think `feature_rules` on `Package` definitely does.

Reply:
These are implicitly defined by the `bt` and `wifi` driver entries, the xtask still sees all these, and more.